release/Jenkinsfile

```diff
@@ -0,0 +1,123 @@
+/**
+ * Pipeline script to be used for releasing BlueOcean
+ */
+node {
+    properties([parameters([
+            choice(choices: 'Vivek Pandey:vivek.pandey@gmail.com\nMichael Neale:mneale@cloudbees.com\nJames Dumay:jdumay@cloudbees.com', description: 'Colon separated Git Username and email.', name: 'GIT_AUTHOR_AND_GIT_EMAIL'),
+            [$class: 'CredentialsParameterDefinition', credentialType: 'com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl', defaultValue: '', description: 'Your GitHub credential', name: 'GITHUB_CREDENTIALS', required: true],
+            [$class: 'CredentialsParameterDefinition', credentialType: 'com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl', defaultValue: '', description: 'Your Jenkins credential', name: 'JENKINS_CREDENTIALS', required: true],
+            string(defaultValue: 'https://github.com/jenkinsci/blueocean-plugin.git', description: 'GIT repo URL', name: 'GIT_REPO_URL'),
+            string(defaultValue: 'master', description: 'GIT branch name to release', name: 'BRANCH')
+    ])])
+
+    deleteDir()
+
+    docker.image('maven:3.3-jdk-8').inside() {
+        def gitNameEmail = (env.GIT_AUTHOR_AND_GIT_EMAIL).split(':')
+        println gitNameEmail[0] + ":" + gitNameEmail[1]
+        String gitName = gitNameEmail[0]
+        String gitEmail = gitNameEmail[1]
+        withEnv(["GIT_COMMITTER_EMAIL=$gitEmail", "GIT_COMMITTER_NAME=$gitName", "GIT_AUTHOR_NAME=$gitName", "GIT_AUTHOR_EMAIL=$gitEmail"]) {
+            withCredentials([
+                    usernamePassword(credentialsId: '${JENKINS_CREDENTIALS}', passwordVariable: 'JENKINS_PASSWORD', usernameVariable: 'JENKINS_USERNAME'),
```

Comment:
Its clear text. If it's running in insecure env, we better keep them encrypted but then we need to tell maven where encryption keys are so anyone who can get to env variable can possibly get to those encryption keys. Something we should run it by Ben?